src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypeCalculationArgumentCollection.java

```diff
@@ -165,4 +170,27 @@ public GenotypeCalculationArgumentCollection( final GenotypeCalculationArgumentC
      */
     @Argument(shortName="ploidy", fullName="sample-ploidy", doc="Ploidy (number of chromosomes) per sample. For pooled data, set to (Number of samples in each pool * Sample Ploidy).", optional=true)
     public int samplePloidy = HomoSapiensConstants.DEFAULT_PLOIDY;
+
+    /**
+     * Apply population priors to genotype likelihoods.  If specified, a population resource VCF must be supplied via the -supporting argument.
+     */
+    @Argument(fullName = "apply-priors", optional = true)
+    public boolean applyPriors = false;
+
+    /**
+     * Supporting external panel. Allele counts from this panel (taken from AC,AN or MLEAC,AN or raw genotypes) will
+     * be used to inform the frequency distribution underlying the genotype priors. These files must be VCF 4.2 spec or later.
+     * Note that unlike CalculateGenotypePosteriors, HaplotypeCaller only allows one supporting callset.
+     */
+    @Argument(fullName="supporting-callsets", shortName = "supporting", doc="Other callsets to use in generating genotype posteriors", optional=true)
+    public FeatureInput<VariantContext> supportVariants = null;
+
+    /**
+     * When a variant is not seen in a panel, this argument controls whether to infer (and with what effective strength)
+     * that only reference alleles were observed at that site. E.g. "If not seen in 1000Genomes, treat it as AC=0,
+     * AN=2000". This is applied across all external panels, so if numRefIsMissing = 10, and the variant is absent in
```

Comment:
I don't love any of the solutions here, but I do think if a variant is absent from all the resources then it should get a higher prior than if it's absent from just one.  I suppose we make the assumption that the sets of samples in the resource files are disjoint and it's certainly not great that we have to apply the same number of each callset. But we can't determine the number of samples from the header because it might be a sites-only file and we can't use AN without traversing the whole VCF.  It would greatly simplify things to limit CGP to one supporting callset as well, but I'm always hesitant to reduce functionality.

Reply:
The thing is, if there are multiple resources the user can take that into account when choosing this value.  What worries me most is the case of a single big resource, like gnomAD, along with a few boutique add-ons for whatever reason.  Then the prior comes almost entirely from the big one and should hardly change when the little ones are added.